webui/modules/implementations/patches/bark_custom_voices.py

```diff
@@ -77,8 +81,26 @@ def generate_course_history(fine_history):
     return fine_history[:2, :]
 
 
+def load_codec_model(use_gpu=True, force_reload=False):
```

Comment:
It's probably a better idea to use the encodec model load function from `webui/modules/implementations/patches/bark_generation.py` to reduce duplicate code. (It would also make sure the model follows the settings provided by the user)

https://github.com/gitmylo/audio-webui/blob/master/webui/modules/implementations/patches/bark_generation.py#L634-L650